Exercise-HTML-CSS-1/index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Ejercicio 1</title>
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="css/styles.css">
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js" integrity="sha384-smHYKdLADwkXOn1EmN1qk/HfnUcbVRZyYmZ4qpPea6sjB/pTJ0euyQp0Mk8ck+5T" crossorigin="anonymous"></script>
+</head>
+<body>
+	<div class="container">
+		<div class="row top">
+		<img src="https://zgab33vy595fw5zq-zippykid.netdna-ssl.com/wp-content/uploads/2017/08/wordmark-tagline-pink-2.png" class="img-fluid" alt="Logo" />
+		</div>
+	</div>
+	<div class="container">
+		<div class="row">
+		<h2>About</h2>
+		</div>
+		<div class="row">
+		<p>It can be intimidating for women to learn and ask questions when they are in an <b>extreme minority</b>. While open and welcoming, today's building developer community is up to <b>91% male</b>. If we can empower more females with the confidence in their technological capabilites <i>we can begin to change this landscape</i>.</p>
+		</div>
+	</div>
+	<div class="container">
+		<div class="row">
+		<h2>Our Locations</h2>
+		</div>
+		<div class="row">
+			<ul>
+				<li>USA
+					<ul>
+						<li>
+							<a href="">New York</a>
+						</li>
+						<li>
+							<a href="">Columbus</a>
+						</li>
+						<li>
+							<a href="">Austin</a>
+						</li>
+						<li>
+							<a href="">Phily</a>
+						</li>
+					</ul>
+				</li>
+
+				<li>World
+					<ul>
+						<li>
+							<a href="">Ottawa</a>
+						</li>
+						<li>
+							<a href="">Sydney</a>
+						</li>
+					</ul>
+				</li>
+			</ul>
+		</div>
+	</div>
+	<div class="container">
+		<div class="row">
+		<h2>Upcoming Classes</h2>
+		</div>
+		<div class="row">
+		<table>
+			<tr>
+				<th>Location</th> 
+				<th>Topic</th>
+			</tr>
+			<tr>
+				<td>New York</td> 
+				<td>JavaScript & jQuery</td>
+			</tr>
+			<tr>
+				<td>San Francisco</td> 
+				<td>HTML & CSS</td>
+			</tr>
+		</table>
+		</div>
+	</div>
+
+	<div class="container">
+		<div class="row">
+			<h2>Gallery</h2>
+		</div>
+		<div class="row">
+			<div class="col">
+				
+			</div>
+		</div>
+
+	</div>
+
+	<div class="container">
+		<div class="row">
+		<h2>Sign Up! </h2>
+		</div>
+		
+			<form >
+				<div class="row">

```

Comment:
Try to add more Bootstrap style to forms
https://getbootstrap.com/docs/4.1/components/forms/